SUTrivBot/Repo/triviaQAData.json

```diff
@@ -1,67 +1,1003 @@
-    },
-    {
-      "questionText": "Give the name of one of the Zoomans (not including Greg and Steven). Bonus for all of them",
-      "answers": ["Y-6", "J-10", "U-12", "F-3"],
-      "answerAll": false,
-      "bonusAnswers": ["Y-6", "J-10", "U-12", "F-3"],
-      "bonusAll": true,
-      "points": 2,
-      "bonusPoints": 5
-    },
-    {
-      "questionText": "What does Buck try to order at The Big Donut in Shirt Club?",
-      "answers": ["Caprese", "salad"],
-      "answerAll": true,
-      "bonusAnswers": [""],
-      "bonusAll": false,
-      "points": 2,
-      "bonusPoints": 0
-    },
-    {
-      "questionText": "What is Steven's canonical birth month? Bonus: name the day too",
-      "answers": ["August"],
-      "answerAll": false,
-      "bonusAnswers": ["August", "15"],
-      "bonusAll": true,
-      "points": 1,
-      "bonusPoints": 3
-    }
-  ]
-}
\ No newline at end of file
+﻿{
+  "questionTemplate": {
+    "questionText": "",
+    "answers": [],
+    "answersRequired": 1,
+    "bonusAnswers": [],
+    "bonusAll": false,
+    "points": 0,
+    "bonusPoints": 0
+  },
+  "questions": [
+    {
+      "questionText": "Why do snakes make Steven cry?",
+      "answerText": "They don't have any arms!",
+      "answers": ["no", "arms"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 1,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "List the full names (if known) of all three Cool Kids",
+      "answerText": "Buck Dewey, Jenny Pizza, Sour Cream",
+      "answers": ["Buck Dewey", "Jenny Pizza", "Sour Cream"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 2,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What is Peridot's Homeworld classification? (Facet or Cut - bonus name both)",
+      "answerText": "Facet-2F5L Cut-5XG",
+      "answers": ["Facet-2F5L", "Cut-5XG"],
+      "answersRequired": 1,
+      "bonusAnswers": ["Facet-2F5L", "Cut-5XG"],
+      "bonusAll": true,
+      "points": 1,
+      "bonusPoints": 2
+    },
+    {
+      "questionText": "Give the name of one of the Zoomans (not including Greg and Steven). Bonus for all of them",
+      "answerText": "Wy-Six, Jay-Ten, U-12, F-3",
+      "answers": ["Y-6", "J-10", "U-12", "F-3"],
+      "answersRequired": 1,
+      "bonusAnswers": ["Y-6", "J-10", "U-12", "F-3"],
+      "bonusAll": true,
+      "points": 2,
+      "bonusPoints": 5
+    },
+    {
+      "questionText": "What does Buck try to order at The Big Donut in Shirt Club?",
+      "answerText": "Caprese salad",
+      "answers": ["Caprese", "salad"],
+      "answersRequired": null,
+      "bonusAnswers": [""],
+      "bonusAll": false,
+      "points": 2,
+      "bonusPoints": 0
+    },
+    {
+      "questionText": "What is Steven's canonical birth month? Bonus: name the day too",
+      "answerText": "2017-08-15T04:00:00.000Z",
+      "answers": ["August"],
```

Comment:
Though parsing a Timestamp answer into a date is something we could add, just wanted to point out again that as it stands this Answer Text would not be accepted.